src/utils/featureFlags.js

```diff
@@ -1,8 +1,11 @@
 // These flags are defined in webpack.config.js
-// and enabled/disabled on the command line
+// and configured on the command line
```

Comment:
Could you explain how we can configure on the command line? For example, if I wanted to test 'Jio' language list locally

Reply:
You would do `LANG_LIST=jio npm run dev`

Reply:
works with my environment